ipl-auction-react/src/control/outLinedTextField.css

```diff
@@ -17,27 +17,28 @@ label {
   pointer-events: none;
 }
 
-input {
+.input {
   font-size: 1rem;
   outline: none;
   border: 2px solid gray;
   border-radius: 5px;
   padding: 1rem 0.7rem;
   color: gray;
   transition: 0.1s ease-out;
+  width: 100%;
 }
 
-input:focus {
+.input:focus {
   border-color: #15327d;
 }
 
-input:focus + label {
+.input:focus + label {
   color: #15327d;
   top: 0;
   transform: translateY(-50%) scale(0.9);
 }
 
-input:not(:placeholder-shown) + label {
+.input:not(:placeholder-shown) + label {
   top: 0;
-  transform: translateY(-50%) scale(0.9);
+  transform: translateY(+50%) scale(0.9);
```

Comment:
Is it working with +50%?

Reply:
yes, the placeholder is an empty string so +50% and -50%  shows same effect